internal/controller/policy_subreconciler_webhook.go

```diff
@@ -77,6 +77,9 @@ func (r *policySubReconciler) reconcileValidatingWebhookConfiguration(
 				AdmissionReviewVersions: []string{"v1"},
 			},
 		}
+		if r.featureGateAdmissionWebhookMatchConditions {
```

Comment:
I could add something like:

```golang
if !r.featureGateAdmissionWebhookMatchConditions && (policy.GetMatchConditions().Size() > 0) {
    log.Info("Skipping setting matchConditions for policy {} as the feature gate is not enabled", policy.GetName())
}
```

Does it seem worth it?

Reply:
Yes, I think it would be nice to show this information to the user. Just go with a `if found { ... } else { info }`

Reply:
Done in https://github.com/kubewarden/kubewarden-controller/pull/814/commits/2aa2120f4a30e93fcf9c8300601693b351a01bbe.